### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Markolie**
Fixes https://github.com/ParadiseSS13/Paradise/issues/5834.
Fixes https://github.com/ParadiseSS13/Paradise/issues/5833.
Fixes https://github.com/ParadiseSS13/Paradise/issues/5828.

Also fixes some minor area issues in maintenance.

### CODE REVIEW

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 1]
```diff
@@ -144,6 +144,9 @@ proc/issyndicate(mob/living/M as mob)
 	var/mob/living/carbon/human/M = synd_mind.current
 	var/obj/item/organ/external/head/head_organ = M.get_organ("head")
 
+	M.mutations = list()
+	M.disabilities = 0
+	M.overeatduration = 0
```
Comment (Markolie) [inline code/game/gamemodes/nuclear/nuclear.dm]: Fun fact: the obesity disability is terribly implemented.

Comment (Plotron) [inline code/game/gamemodes/nuclear/nuclear.dm]: Snowflaek fat people!

I always knew I was a special snowflake IRL... :D

Comment (Crazylemon64) [inline code/game/gamemodes/nuclear/nuclear.dm]: The `disabilities` bitfield is in general really bad - you might want to use the `CureATTRIBUTE` functions instead to ensure things like blindness and similar instantly update, and I think there's also a `becomeSlim` proc that will make someone normal, and also make their torso organ normal as well. (though I think `becomeSlim` gives a message, too, so perhaps a silence parameter would be wise)

Comment (Markolie) [inline code/game/gamemodes/nuclear/nuclear.dm]: Thanks - once those changes go through I'll update these checks respectively.

Comment (Crazylemon64) [inline code/game/gamemodes/nuclear/nuclear.dm]: @Markolie The procs I mentioned are already a thing with #5601 and #5479

Comment (Markolie) [inline code/game/gamemodes/nuclear/nuclear.dm]: I had another look, and I think this is the most global solution without having to manually remove every disability manually with their procs. This also gives a message, but I don't think it's worth implementing a silent var just for this. 

If at some point we get datum antags this should probably become a generalized proc for randomizing antags.

**Pull request merged**